packages/emotion/types/index.d.ts

```diff
@@ -1,17 +1,28 @@
 // Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
-// TypeScript Version: 2.3
+// TypeScript Version: 2.8
```

Comment:
What's the point of these comments?

Reply:
@mitchellhamilton dtslint parse these header comments, and check correctness of type for the version and above.